src/test/run-make/nvptx-binary-crate/Makefile

```diff
@@ -0,0 +1,9 @@
+-include ../../run-make-fulldeps/tools.mk
+
+ifeq ($(TARGET),nvptx64-nvidia-cuda)
+all:
+	$(RUSTC) main.rs -Clink-arg=--arch=sm_60 --crate-type="bin" -O --target $(TARGET)
```

Comment:
Does `RUSTFLAGS=-C target-cpu=sm_60` achieve the same effect here as passing the link flags ?

Reply:
Thanks, that's a nice catch.
I've addressed this providing `target_cpu` via `--fallback-arch` arg to the linker in the latest commit.